index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Bootstrap CDN -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous" />
+    <!-- Bootstrap Icons CDN -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css" />
+    <link rel="stylesheet" href="style.css">
+    <title>Capstone 1</title>
+</head>
+
+<body>
+
+    <div class="container-fluid m-0 vh-100">
+        <header class="py-2">
+            <nav><img class="h-30 ps-3 mobilemenu" src="img/hamburguer.svg" alt="hamburguer menu">
+                <div class="social">
+                    <ul class="d-flex justify-content-end">
+                        <li><a href="https://github.com/gealsanchez"><img src="img\github.svg" alt="logo github"></a></li>
+                        <li><a href="https://www.linkedin.com/in/gerson-sanchez-88309b57/"><img
+                                    src="img\Linkedin icon.svg" alt="logo linkedin"></a></li>
+                        <li><a href="https://angel.co/"><img src="img\angel list.svg" alt="logo angel list"></a></li>
+                        <li><a href="https://twitter.com/gealsanchez"><img src="img\twitter.svg" alt="logo twitter"></a></li>
+                        <li> <a href="https://medium.com/tag/javascript"><img src="img\medium.svg" alt="logo medium"></a></li>
+                    </ul>
+                </div>
```

Comment:
- [ ] Kindly try to follow the design guidelines as much as possible. For the top navbar, it should be dark and it should have "english, my page, logout" and 2 icons (facebook and twitter).

## Your design:
![image](https://user-images.githubusercontent.com/50721479/198693738-5359c06e-da80-48ca-9147-42464ed573ef.png)

## Design guidelines:
![image](https://user-images.githubusercontent.com/50721479/198693816-84c81fdd-4cca-4072-ad36-8d731bbb286f.png)